README.md

```diff
@@ -32,3 +53,35 @@ var v3 = new Vector(3, 4)
 console.log(v3.getLength());
 // => 5
 ```
+```
+class Vector{
+  constructor(x,y){
+  this.x = x;
+  this.y = y;
+}
+
+plus(vec){
+  let sum1 = this.x + vec.x;
+  let sum2 = this.y + vec.y;
+  return `x:${sum1}, y:${sum2}`
```

Comment:
We want to return a new Vector:
```
return new Vector(sum1, sum2)
```
Alternatively, you don't _need_ to create new variables and could just return:
``` 
return new Vector(this.x + vec.x)
```